src/plugins/dashboard/public/application/top_nav/options.tsx

```diff
@@ -80,6 +90,17 @@ export class OptionsMenu extends Component<Props, State> {
             data-test-subj="dashboardPanelTitlesCheckbox"
           />
         </EuiFormRow>
+
+        <EuiFormRow>
+          <EuiSwitch
+            label={i18n.translate('dashboard.topNav.options.syncColorsBetweenPanelsSwitchLabel', {
+              defaultMessage: 'Sync colors across panels',
```

Comment:
I was initially confused when I saw that Lens wasn't synced with Visualize automatically. Maybe we can name this "Sync color palettes across panels" to show that it's per palette?

Reply:
I think that's a good improvement however I can see that being confusing as well because it could also mean the palettes themselves are synchronized, not the colors (like all panels using the same palette).